https://github.com/openlayers/openlayers/releases/download/release-2.13.1/OpenLayers-2.13.1.tar.gz is gone [PR is waiting]

https://github.com/openlayers/openlayers/releases/download/release-2.13.1/OpenLayers-2.13.1.tar.gz gives for me 404 Github page also in the browser.

As result `make dependencies-get` mentioned in the readme is failing.

```
mateusz@Grisznak:~/Desktop/tmp/opening_hours_map$ make dependencies-get
wget --no-clobber -O js/OpenLayers-2.13.1.tar.gz https://github.com/openlayers/openlayers/releases/download/release-2.13.1/OpenLayers-2.13.1.tar.gz
--2017-05-15 07:46:43--  https://github.com/openlayers/openlayers/releases/download/release-2.13.1/OpenLayers-2.13.1.tar.gz
Resolving github.com (github.com)... 192.30.253.112, 192.30.253.113
Connecting to github.com (github.com)|192.30.253.112|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-05-15 07:46:43 ERROR 404: Not Found.

make: [js/OpenLayers-2.13.1/OpenLayers.js] Error 8 (ignored)
tar -xzf js/OpenLayers-2.13.1.tar.gz -C js/

gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now
make: *** [js/OpenLayers-2.13.1/OpenLayers.js] Error 2

```

--- a/Makefile
+++ b/Makefile
@@ -33,7 +33,7 @@ dependencies-get: js/OpenLayers-$(OpenLayersVersion)/OpenLayers.js
 	cd opening_hours.js/ && git verify-commit HEAD && git submodule update --init --recursive
 
 js/OpenLayers-$(OpenLayersVersion)/OpenLayers.js:
-	-wget --no-clobber -O js/OpenLayers-$(OpenLayersVersion).tar.gz https://github.com/openlayers/openlayers/releases/download/release-$(OpenLayersVersion)/OpenLayers-$(OpenLayersVersion).tar.gz
+	-wget --no-clobber -O js/OpenLayers-$(OpenLayersVersion).tar.gz https://github.com/openlayers/ol2/releases/download/release-$(OpenLayersVersion)/OpenLayers-$(OpenLayersVersion).tar.gz
 	tar -xzf js/OpenLayers-$(OpenLayersVersion).tar.gz -C js/
 
 .PHONY: deploy-on-all-servers

